pyproject.toml

```diff
@@ -27,8 +27,8 @@ requires-python = ">=3.6"
 [project.urls]
 homepage = "https://github.com/ihmeuw-msca/pydisagg"
 
-[tool.setuptools.packages]
-find = {} 
+[tool.setuptools.packages.find]
+where = ["src"]
```

Comment:
I think we can remove this specification, in
https://setuptools.pypa.io/en/latest/userguide/pyproject_config.html
It mentioned that

> When both py-modules and packages are left unspecified, setuptools will attempt to perform [Automatic discovery](https://setuptools.pypa.io/en/latest/userguide/package_discovery.html#auto-discovery), which should cover most popular project directory organization techniques, such as the [src-layout](https://setuptools.pypa.io/en/latest/userguide/package_discovery.html#src-layout) and the [flat-layout](https://setuptools.pypa.io/en/latest/userguide/package_discovery.html#flat-layout).